### PR TITLE
initial deletion 

### PR DESCRIPTION
**Pull request by dderancourt**
deletion of some file generate by the CRA, insertion of a reset to delete initial style css

### CODE REVIEW

**Review by laurentmahieu — DISMISSED** [review]

**Review by laurentmahieu — APPROVED** [review]
ok

File: frontend/package.json [hunk 1]
```diff
@@ -5,7 +5,12 @@
   "dependencies": {
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.2.0"
+    "react-redux": "^7.1.3",
```
Comment (LoicBrassart) [inline frontend/package.json:8]: I'd recommend to install only libraries you need on an *immediate* basis, but that's just a piece of advice

File: frontend/src/reducer/reducer.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import initialState from "../store/store";
```
Comment (LoicBrassart) [inline frontend/src/reducer/reducer.js:1]: This file is totally useless, what do you plan on adding in it ?

**Review by LoicBrassart — APPROVED** [review]
"Initial deletion" adds several logos... Not a real problem, but an incoherence

**Pull request merged**